route/pool.go

```diff
@@ -384,7 +384,9 @@ func (p *EndpointPool) Endpoints(logger *slog.Logger, initial string, mustBeStic
 		logger.Debug("endpoint-iterator-with-round-robin-lb-algo")
 		return NewRoundRobin(logger, p, initial, mustBeSticky, azPreference == config.AZ_PREF_LOCAL, az)
 	default:
-		logger.Error("invalid-pool-load-balancing-algorithm", slog.String("poolLBAlgorithm", p.LoadBalancingAlgorithm))
+		logger.Error("invalid-pool-load-balancing-algorithm",
+			slog.String("poolLBAlgorithm", p.LoadBalancingAlgorithm),
+			slog.Group("data", slog.String("Host", p.host)))
```

Comment:
Sorry, I made a bad suggestion yesterday.

The additional `slog.*` data in the `logger.Error` is already wrapped in a `data` group by default. Correcting the suggestion. But please test and make sure that the log looks reasonable, i.e. something like this:

```
{"log_level": 1, "message": "invalid-pool-load-balancing-algorithm", ...,
 "data": { "Host":"the.pool.host", "poolLBAlgorithm", "round-robin" } }
```

You may need to undo your fix temporarily in the test environment and force the condition again with the instance-pinned request.

```suggestion
			slog.String("Host", p.host))
```